docs/system-info/mirrir-services-and-rates.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
fix: please remove - this is duplicative of the [Box link](https://wustl.box.com/s/umzho1jlyoxj2agbuee77ic8xkk2z4jw), and it would make it public to everyone - even outside the university